app/javascript/components/lease_ups/LeaseUpApplicationsPage.js

```diff
@@ -101,26 +101,6 @@ const LeaseUpApplicationsPage = () => {
     setBulkCheckboxesState({ ...bulkCheckboxesState, [appId]: !bulkCheckboxesState[appId] })
   }
 
-  const performAsyncRequest = ({
```

Comment:
nice! glad you got rid of this function, was very unnecessary obfuscation. What was actually causing the bug @james-wills-sf ?